ert_shared/storage/storage_api.py

```diff
@@ -232,7 +238,8 @@ def get_ensemble(self, ensemble_id):
                     ],
                 }
             )
-        return return_schema
+            return return_schema  
```

Comment:
Remove extra white space and indentation

Reply:
if you have installed `black` you could do black ert_shared/storage/storage_api.py and this will format the file for you removing white-spaces and fix intendation. 

you install by `pip install black` in your python environment. We usually do black on all new code and files to enable everyone to use same formatting. On old code we cannot do this because the changeset's get's very big with only formatting. If we do it, be create a total separate commit with only formatting, and then do our changes in a new commit.